app/views/admin/documents/index.html.erb

```diff
@@ -24,7 +24,11 @@
           <% end %>
         </td>
         <td class="govuk-table__cell"><%= link_to document.searchable.slug, document.searchable %></td>
-        <td class="govuk-table__cell"><%= document.searchable_type %></td>
+        <% if document.searchable_type == 'Journey' and !document.published? %>
```

Comment:
Toto je divna podmienka - nerozumiem jej. 

1. `and` nepouzivajme v ruby sa to zvykne pouzivat v inych kontextoch https://avdi.codes/using-and-and-or-in-ruby/
2.  ta podmienka `published?` je zvlastna. ked to teda zverejnim tak to je ok a nebude to cervene? bol som v tom, ze blank navody maju byt vzdy zvyraznene.

Reply:
Ten and teda zmením na `&&`. `published?` je v journey stav, že už je hotová a môže sa zverejniť. Je to iný stav k blank. To, či je reálne na stránke, hovorí `featured`. Blank bude vždy červený.

Reply:
podľa mňa je výhodnejšie to mať takto všeobecnejšie, že všetko okrem ok published chceme zvýrazniť. Dnes to je jedine blank, ale ak by niekedy pribudol iný nehotový stav, netrebalo by nič meniť. Ale ak chceš, vytvorím tam nejakú metódu `is_blank` alebo dačo.

Reply:
Ok zmenme to teda na && a moze byt.